src/fiatconnect/fiatAccountSchemas/index.test.ts

```diff
@@ -0,0 +1,68 @@
+import { FiatAccountSchema, FiatAccountType } from '@fiatconnect/fiatconnect-types'
+import { getSchema } from 'src/fiatconnect/fiatAccountSchemas'
+import { getAccountNumberSchema } from 'src/fiatconnect/fiatAccountSchemas/accountNumber'
+import { getIbanNumberSchema } from 'src/fiatconnect/fiatAccountSchemas/ibanNumber'
+import { FiatAccountSchemaCountryOverrides } from 'src/fiatconnect/types'
+
+jest.mock('src/fiatconnect/fiatAccountSchemas/accountNumber', () => ({
+  getAccountNumberSchema: jest.fn(),
+}))
+
+jest.mock('src/fiatconnect/fiatAccountSchemas/ibanNumber', () => ({
+  getIbanNumberSchema: jest.fn(),
+}))
```

Comment:
```suggestion
jest.mock('src/fiatconnect/fiatAccountSchemas/accountNumber')
jest.mock('src/fiatconnect/fiatAccountSchemas/ibanNumber')
```
you can drop this mock return value, this comes by default

Reply:
Actually you may want to return something custom here to assert the return value of getSchema

Reply:
Ah asserting the return value is a good idea